docs/index.rst

```diff
@@ -0,0 +1,90 @@
+PreREISE
+=========
+This page is dedicated to the collection, processing and formatting of data that are
+fed to the simulation engine. PreREISE is an open source package written in Python that
+is available on `GitHub <https://github.com/Breakthrough-Energy/PreREISE>`_.
+
+The generation of input data rely on the grid model. For this reason, you must have
+`PowerSimData <https://github.com/Breakthrough-Energy/PowerSimData>`_ cloned in a
+folder adjacent to your clone of PreREISE.
+
+
+General Comments
+----------------
+Profiles are CSV (Comma Separated Values) files enclosing a series of data points
+indexed in time orders. The simulation engine expects a demand, hydro, solar and wind
+profile. All the profiles have UTC timestamp as indices. Columns are the zone IDs and
+plant IDs for the demand and hydro/solar/wind profiles, respectively. Values are load in
+MWh in each zone for the demand and energy generated by a 1MW generator for hydro,
+solar and wind profiles.
+
+The following naming convention is used for these files: **[type]_v[D]**, where **type**
+specifies the profile type (demand, hydro, solar or wind) and **D** refers to the date
+the file has been created. Month (3 letters, e.g., Jan) and year (4 digits, e.g., 2021)
+are used for the date. If two or more profiles of the same type are created the same
+month of the same year, the different versions will be differentiated using a lowercase
+letter added right after the year, e.g, ``demand_vJan2021.csv`` (first),
+``demand_vJan2021b.csv`` (second), ``demand_vJan2021c.csv`` (third) and so forth.
+
+Note that input data are publicly available for the Breakthrough Energy Sciences (BES)
+grid model (see `publication <https://arxiv.org/abs/2002.06155>`_). You don't need to
+generate any input data if you use the scenario framework to carry out power flow study.
```

Comment:
The fact that no input data needs to be generated (i.e. `PowerSimData` has input data built-in) by the user to use the scenario framework seems like important information, especially considering the flow-chart on the main docs page has `PreReise` as the starting point. Should this point be highlighted or higher up?

Reply:
Yes it should. 

Also, it will be mentioned in the new README (see https://github.com/Breakthrough-Energy/PreREISE/tree/ben/README, #195)

Reply:
It looks like this now:
<img width="833" alt="Screen Shot 2021-07-20 at 10 30 31 AM" src="https://user-images.githubusercontent.com/16549571/126369172-af7a517d-e52b-4166-af4e-d933a3c01ce1.png">

Reply:
@BainanXia, the note will look like this on the website

Reply:
I like it and hope people won't miss it.

Reply:
I can move it at the top of the document, right before the General Comments section.